packages/contracts/src/dollar/core/UbiquityDollarToken.sol

```diff
@@ -155,7 +62,6 @@ contract UbiquityDollarToken is ERC20Ubiquity {
         uint256 amount
     ) internal override {
         super._transfer(sender, recipient, amount);
```

Comment:
The `_transfer()` method can also be removed as far as I understand since there is no need to override it anymore

Reply:
[This method](https://github.com/ubiquity/ubiquity-dollar/blob/bb6543eb61e8f328a9b12d20735d4785e9570c2b/packages/contracts/src/dollar/core/UbiquityDollarToken.sol#L59) can be removed since it doesn't modify the parent's [method](https://github.com/ubiquity/ubiquity-dollar/blob/bb6543eb61e8f328a9b12d20735d4785e9570c2b/packages/contracts/src/dollar/core/ERC20Ubiquity.sol#L191). Hence when `UbiquityDollarToken.transfer()` is called [this one](https://github.com/ubiquity/ubiquity-dollar/blob/bb6543eb61e8f328a9b12d20735d4785e9570c2b/packages/contracts/src/dollar/core/ERC20Ubiquity.sol#L191) will be used.

Reply:
fixed in https://github.com/ubiquity/ubiquity-dollar/pull/875/commits/fc32be5a4676b0a0833b070196940cdd85a3d0ef

Reply:
yeah missed that it was `internal`